crates/bundler/src/bundler.rs

```diff
@@ -78,4 +101,142 @@ impl Bundler {
 
         Ok(tx_hash)
     }
+
+    // TODO: add more relay endpoints support
+    /// Send a bundle as Flashbots bundles
+    #[allow(clippy::needless_return)]
+    #[allow(clippy::clone_double_ref)]
+    pub async fn send_next_bundle_flashbots(&self, uos: &Vec<UserOperation>, test: Option<bool>) -> anyhow::Result<H256> {
+
+        let mut relay_endpoint: &str = RELAY_ENDPOINTS[0].1;
+        if Some(true) == test {
+            relay_endpoint = RELAY_ENDPOINTS[1].1; 
+        };
+        
+
+        if uos.is_empty() {
+            info!("Skipping creating a new bundle, no user operations");
+            return Ok(H256::default());
+        };
+
+        info!("Creating a new bundle with {} user operations", uos.len());
+        trace!("Bundle content: {uos:?}");
+
+        let provider = Provider::<Http>::try_from(self.eth_provider_address.clone())?;
+
+        let _bundle_signer = env::var("FLASHBOTS_IDENTIFIER").expect("FLASHBOTS_IDENTIFIER environment variable is not set");
+
+        let bundle_signer = _bundle_signer.parse::<LocalWallet>()?;
+
+        let mut fb_middleware = FlashbotsMiddleware::new(
+                    provider.clone(),
+                    Url::parse(relay_endpoint.clone())?,
+                    bundle_signer.clone(),
+                );
+        fb_middleware.set_simulation_relay(Url::parse(relay_endpoint.clone()).unwrap());
+
+        let client = Arc::new(
+            SignerMiddleware::new(
+                fb_middleware,
+                self.wallet.signer.clone(),
+            )
+        );
+
+        let ep = EntryPointAPI::new(self.entry_point, client.clone());
+
+        let nonce = client
```

Comment:
some code is repeated in this function and normal send bundle, we could move some logic into new functions